npm postinstall Error: ENOENT

installing latest releases, after `0.5.18` are giving me the following error:
`Running post install.js Error running postinstall script { Error: ENOENT: no such file or directory, unlink '/Users/mystic/work/react/mysteem/node_modules/steem/index.js' at Object.fs.unlinkSync (fs.js:1053:18) at Object.<anonymous> (/Users/mystic/work/react/mysteem/node_modules/steem/scripts/post-install.js:14:16)
npm -v 5.0.4
node -v v8.0.0`

tried, it throws this 
`mysteem_tmp cnpm install steem@0.5.20-beta.1 --save


> steem@0.5.20-beta.1 postinstall /Users/mystic/work/react/mysteem_tmp/node_modules/steem
> cross-env scripts/post-install.js

module.js:487
    throw err;
    ^

Error: Cannot find module '..'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/mystic/work/react/mysteem_tmp/node_modules/.bin/cross-env:4:16)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)`

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "steem",
-  "version": "0.5.19",
+  "version": "0.5.20",
   "description": "Steem.js the JavaScript API for Steem blockchain",
   "main": "lib/index.js",
   "scripts": {

--- a/scripts/index.rn.js
+++ b/scripts/index.rn.js
@@ -17,5 +17,5 @@ global.crypto = {
   },
 };
 
-require('./dist/steem.min.js');
+require('../dist/steem.min.js');
 module.exports = global.steem;

--- a/scripts/post-install.js
+++ b/scripts/post-install.js
@@ -7,7 +7,7 @@ try {
     const path = require('path');
     if (packageJSON.dependencies['react-native']) {
         const fs = require('fs');
-        const index = path.join(__dirname, '..', 'index.js');
+        const index = path.join(__dirname, '..','lib', 'index.js');
         const RNIndex = path.join(__dirname, 'index.rn.js');
         const indexContent = fs.readFileSync(RNIndex, { encoding: 'utf8' })
         if (indexContent) {

